SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -310,11 +316,14 @@ void CFlowCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
   SetVolumeOutputValue("DENSITY",    iPoint, Node_Flow->GetSolution(iPoint, 0));
   SetVolumeOutputValue("MOMENTUM-X", iPoint, Node_Flow->GetSolution(iPoint, 1));
   SetVolumeOutputValue("MOMENTUM-Y", iPoint, Node_Flow->GetSolution(iPoint, 2));
+  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetSolution(iPoint, 3));
```

Comment:
```suggestion
  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetPrimitive(iPoint, 1));
```
For compressible flows, the velocity is stored in the `Primitive` data, not the `Solution` data. The indices for the x,y, and z velocity components in the `Primitive` data are 1,2, and 3 respectively in the `CEulerVariable` class.

Reply:
This is implemented as well